src/components/Verse/OverflowVerseActionsMenuBody.tsx

```diff
@@ -93,9 +86,7 @@ const OverflowVerseActionsMenuBody: React.FC<Props> = ({ verse }) => {
 
       <VerseActionAdvancedCopy verse={verse} />
 
-      <PopoverMenu.Item onClick={onTafsirsClicked} icon={<TafsirIcon />}>
-        {t('quran-reader:tafsirs')}
-      </PopoverMenu.Item>
+      <TafsirModal verse={verse} />
```

Comment:
probably not a very good name. Since this is actually the TafsirModal + PopoverMenuItem to trigger it

Reply:
TafsirVerseAction

Reply:
Pass only the props that you need